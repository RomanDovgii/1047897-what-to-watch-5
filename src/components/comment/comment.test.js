import React from "react";
import renderer from "react-test-renderer";
import Comment from "./comment";

const testComment = {
  id: 1,
  user: {
    id: 4,
    name: `Kate Muir`
  },
  rating: 8.9,
  comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
  date: `2019-05-08T14:13:56.569Z`
};

it(
    `Should render correct comment`,
    () => {
      const tree = renderer
      .create(
          <Comment
            commentData={testComment}
          />
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);
