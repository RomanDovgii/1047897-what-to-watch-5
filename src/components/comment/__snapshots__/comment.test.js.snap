// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct comment 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Kate Muir
      </cite>
      <time
        className="review__date"
        dateTime="2019-05-08T14:13:56.569Z"
      >
        May
         
        8
        , 
        2019
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8,9
  </div>
</div>
`;
